packages/odyssey-react-mui/src/OdysseyProvider.tsx

```diff
@@ -33,11 +34,15 @@ const OdysseyProvider = ({
   children,
   nonce,
   themeOverride,
+  tokenOverride,
   languageCode,
   translationOverrides,
 }: OdysseyProviderProps) => (
   <OdysseyCacheProvider nonce={nonce}>
-    <OdysseyThemeProvider themeOverride={themeOverride}>
+    <OdysseyThemeProvider
+      themeOverride={themeOverride}
+      tokenOverride={tokenOverride}
```

Comment:
I'd like to rename `tokenOverride` to `tokensOverride` since you're overriding the tokens, not a single token. Also, if we named it `tokenOverrides`, I'd expect it to be an array, so it makes sense to be `tokensOverride`:

```suggestion
      tokensOverride={tokensOverride}
```